public/scss/modules/_photo.scss

```diff
@@ -19,7 +19,22 @@ Photo
   width: 0.5%;
 }
 .grid-sizer, .photo-grid-item {
-  width: 24.5%;
+
+  width: 49%;
+  // Medium devices (tablets, 768px and up)
+  @media (min-width: 768px) {
```

Comment:
There are beautifully defined variables in `_variables.scss` so that you don't have to hardcode these breakpoints :)